### ISSUE TITLE
Bad results when inference

### ISSUE THREAD

**Issue by Liujingxiu23** [thuhcsi/NeuFA#3]
Hi, thank you for your work and sharing!
I tried the model,  the model structure is the original version, except that I used 80-dim mels with hop_size=256 without normalization instead of mfcc.
The loss seems good. But the inference results are bad, the result for all phones is [0.01, 0.01].
I checked the "w1" in inference results, it is good, it is diagonal.
The "boundary" result in inference seems:
![image](https://user-images.githubusercontent.com/49260794/184573464-71867b13-2406-4a89-a68a-5bae9997bef5.png)
Dose the loss good?
![image](https://user-images.githubusercontent.com/49260794/184569778-75990b51-b6b0-48ba-8011-86a3ca289aa2.png)
![image](https://user-images.githubusercontent.com/49260794/184572276-e1a90100-023f-4759-ab88-df0d93d7e3c7.png)
I did not know where is wrong

**Comment by Liujingxiu23** [comment]
I find that I only do "pretrain" without "boundary loss" related. What should I do if I do not have any "phoneme/word bounday" of any of my data, and can not do dev/semi/semi2 training？

**Comment by petronny** [comment]
Well, to use NeuFA, you have to use some data with boundaries to train the boundary detector.

For Chinese, you can use [the Chinese Standard Mandarin Speech Copus](https://www.data-baker.com/open_source.html) from Databaker.

**Comment by Liujingxiu23** [comment]
Thank you for your reply, I got it.
